includes/wc-template-functions.php

```diff
@@ -2651,7 +2651,7 @@ function woocommerce_form_field( $key, $args, $value = null ) {
 
 				} elseif ( ! is_null( $for_country ) && is_array( $states ) ) {
 
-					$field .= '<select name="' . esc_attr( $key ) . '" id="' . esc_attr( $args['id'] ) . '" class="state_select ' . esc_attr( implode( ' ', $args['input_class'] ) ) . '" ' . implode( ' ', $custom_attributes ) . ' data-placeholder="' . esc_attr( $args['placeholder'] ) . '">
+					$field .= '<select name="' . esc_attr( $key ) . '" id="' . esc_attr( $args['id'] ) . '" class="state_select ' . esc_attr( implode( ' ', $args['input_class'] ) ) . '" ' . implode( ' ', $custom_attributes ) . ' data-placeholder="' . esc_attr( $args['placeholder'] ? $args['placeholder'] : esc_html__( 'Select a state&hellip;', 'woocommerce' ) ) . '">
```

Comment:
<img width="445" alt="screen shot 2019-02-01 at 7 32 46 am" src="https://user-images.githubusercontent.com/7317227/52132329-c55d0880-25f3-11e9-87e8-652f8b4db7f8.png">

This outputs a "Select a state" placeholder for provinces.

Reply:
Resolved